docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -39,406 +54,385 @@ The first output line should look like
    :width: 100%
 
-   Graph with all inputs (data, circles; and code, diamond) to
-   the Quantum Espresso calculation (square) that you will create in
-   :ref:`calculations` section of this tutorial.
+   Graph with all inputs (data, circles; and code, diamond) to the Quantum ESPRESSO calculation (square) that you will create in the :ref:`calculations` section of this tutorial.
 
 .. _fig_graph:
 .. figure:: include/images/verdi_graph/batio3/graph-full.png
    :width: 100%
 
-   Same as above, but also with the outputs that the daemon will create and
-   connect automatically.
-   The RemoteData node is created during submission and can be thought as a
-   symbolic link to the remote folder in which the calculation runs on the
-   cluster. The other nodes are created when the calculation has finished,
-   after retrieval and parsing. The node with linkname “retrieved” contains
-   the raw output files stored in the AiiDA repository; all other nodes are
-   added by the parser. Additional nodes (symbolized in gray) can be added
-   by the parser (e.g., an output StructureData if you performed a
-   relaxation calculation, a TrajectoryData for molecular dynamics, …).
+   Same as :numref:`fig_graph_input_only`, but also with the outputs that the engine will create and connect automatically.
+   The ``RemoteData`` node is created during submission and can be thought as a symbolic link to the remote folder in which the calculation runs on the cluster.
+   The other nodes are created when the calculation has finished, after retrieval and parsing.
+   The node with linkname 'retrieved' contains the raw output files stored in the AiiDA repository; all other nodes are added by the parser.
+   Additional nodes (symbolized in gray) can be added by the parser (e.g., an output ``StructureData`` if you performed a relaxation calculation, a ``TrajectoryData`` for molecular dynamics etc.).
 
-You can create a similar graph for any calculation node by using the
-utility ``verdi graph generate <pk>/<uuid>``. For example, before you obtained
-information (in text form) for ``UUID = ce81c420``. To visualize similar
-information in graph(ical) form, run (replacing ``<pk>`` or ``<uuid>`` with the
-PK or UUID of the node respectively):
+You can create a similar graph for any calculation node by using the utility ``verdi graph generate <IDENTIFIER>``.
+For example, before you obtained information (in text form) for UUID ``ce81c420`` using the command ``verdi process show``.
+To visualize similar information in graph(ical) form, run the command:
 
 .. code:: console
 
-    verdi graph generate <pk>/<uuid>
+    verdi graph generate <IDENTIFIER>
 
-This command will create the file ``<pk>.dot`` that can be rendered by
-means of the utility ``dot``. If you now type
+This command will create the file ``<PK>.dot`` that can be rendered by means of the utility ``dot`` as follows:
 
 .. code:: console
 
-    dot -Tpdf -o <pk>.pdf <pk>.dot
+    dot -Tpdf -o <PK>.pdf <PK>.dot
```

Comment:
I guess these PKs should then be replaced as well?

Reply:
I am not sure, because the `verdi graph generate` tool will print `<PK>.dot` even if you passed a UUID to the command.

Reply:
ok